core/src/main/java/hudson/Functions.java

```diff
@@ -233,6 +233,8 @@ public static void initPageVariables(JellyContext context) {
 
         // The path starts with a "/" character but does not end with a "/" character.
         context.setVariable("rootURL", rootURL);
+        Jenkins jenkins = Jenkins.getActiveInstance();
```

Comment:
:bug: I suspect it may impact Jenkins startup. IIRC Jenkins is null when we open "Jenkins is loading page", but <code> initPageVariables</code> is being invoked for any layout